### ISSUE TITLE
Nest uiowa_alerts.settings configuration as collection

### ISSUE THREAD

**Issue by richardbporter** [uiowa/uiowa#1720]
Not sure why I underscored it but it should be a collection:
```
hawk_alert:
  display: true
  source: ...
custom_alert:
  display: false
...
```
Requires a DB update to migrate existing config.

**Comment by richardbporter** [comment]
Received confirmation that the ability to toggle Hawk Alert display should be removed and set to always on.

**Comment by richardbporter** [comment]
Related to #1394

**Comment by richardbporter** [comment]
Not sure why this wasn't closed with #1806.
